proto/message_contents/private_key.proto

```diff
@@ -11,17 +11,16 @@ import "message_contents/public_key.proto";
 
 option go_package = "github.com/xmtp/proto/go/message_contents";
 
-
 // PrivateKey generalized to support different key types
-message PrivateKey {
+message SignedPrivateKey {
```

Comment:
The diff here is BS, I simply swapped the legacy types and the V2 types, to move the latter to the top. No actual changes.